great_expectations/cli/datasource.py

```diff
@@ -478,9 +487,16 @@ def _collect_postgres_credentials(default_credentials={}):
     credentials["username"] = click.prompt("What is the username for the postgres connection?",
                             default=default_credentials.get("username", "postgres"),
                             show_default=True)
-    credentials["password"] = click.prompt("What is the password for the postgres connection?",
-                            default="",
-                            show_default=False, hide_input=True)
+    # This is a minimal workaround we're doing to deal with hidden input problems using Git Bash on Windows
+    # TODO: Revisit this if we decide to fully support Windows and identify if there is a better solution
+    if _is_windows_platform():
+        credentials["password"] = click.prompt("What is the password for the postgres connection?",
+                                               default="",
+                                               show_default=False)
+    else:
+        credentials["password"] = click.prompt("What is the password for the postgres connection?",
+                                               default="",
+                                               show_default=False, hide_input=True)
```

Comment:
This could be made much simpler if you swap the helper function to something like `_should_hide_input()`. Then no logic and code dupe is needed

```
credentials["password"] = click.prompt(
    "What is the password for the postgres connection?",
    default="",
    show_default=False,
    hide_input=_should_hide_input()
)
```